openstef/pipeline/create_forecast.py

```diff
@@ -38,26 +39,25 @@ def create_forecast_pipeline(
     Args:
         pj (PredictionJobDataClass): Prediction job
         input_data (pd.DataFrame): Training input data (without features)
+        mlflow_tracking_uri (str): MlFlow tracking URI
         trained_models_folder (Path): Path where trained models are stored
```

Comment:
The mlflow_tracking_uri is either the url to the mlflow service or the path to the metadata store (can be file-based or database). The artifict_root/trained_models_folder is always the path to the artifact store (file-based only) to be able to remove artifacts. I'll check if the artifact_root can be deduced from the metadata store/mlflow service to avoid having to give 2 arguments. Other option to make it simpler is to only pass trained_models_folder to the remove_models function instead of to the class/init.

Reply:
Probably it's nicer to only pass trained_models_folder to remove_models since metadata store does not always have info where artifact_root is. Will implement that to clean it up